recipes/diligent-core/all/conanfile.py

```diff
@@ -169,6 +173,11 @@ def package_info(self):
         self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Graphics", "GraphicsTools", "interface"))
         self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Graphics", "HLSL2GLSLConverterLib", "interface"))
 
+        if self.settings.os == 'Windows':
+            self.cpp_info.defines.append("FILE2STRING_PATH=${CONAN_BIN_DIRS_DILIGENT-CORE}/File2String.exe")
+        else:
+            self.cpp_info.defines.append("FILE2STRING_PATH=${CONAN_BIN_DIRS_DILIGENT-CORE}/File2String")
```

Comment:
```suggestion
```
Alternate solution, because FILE2STRING_PATH seems to be a cmake variable, and not a C++ macro, cf https://github.com/DiligentGraphics/DiligentCore/search?q=FILE2STRING_PATH

Reply:
@AndreyMlashkin are you sure FILE2STRING_PATH is supposed to be a c++ macro, as opposed to a cmake variable ?